docs/quick-start/configuration/page.md

```diff
@@ -11,36 +11,37 @@ Follow this directory structure within the GoFr project:
 ```dotenv
 my-gofr-app/
 ├── configs/
-│   ├── dev.env
-│   ├── staging.env
-│   └── prod.env
+│   ├── .local.env
+│   ├── .dev.env
+│   ├── .staging.env
+│   └── .prod.env
 ├── main.go
 └── ...
 ```
 
 By default, GoFr starts HTTP server at port 8000, in order to change that we can add the config `HTTP_PORT`
 Similarly to Set the app-name user can add `APP_NAME`. For example:
 
-```bash
+```dotenv
 # configs/.env
 
 APP_NAME=test-service
 HTTP_PORT=9000
 ```
 
 ## Configuring Environments in GoFr
-GoFr uses an environment variable, `APP_ENV`, to determine the application’s current environment. This variable also guides GoFr to load the corresponding environment file.
+GoFr uses an environment variable, `APP_ENV`, to determine the application`s current environment. This variable also guides GoFr to load the corresponding environment file.
```

Comment:
In `application's`, an apostrophe (') is correctly used, so it is not required to update it with a quote (`). Please revert